Library/Formula/delta.rb

```diff
@@ -0,0 +1,51 @@
+require "formula"
+
+# Two requests are needed to download from Tigris.org. In the first, the server
+# provides a cookie and then redirects the client to a servlet. The servlet
+# requires this cookie, or it will send the client in a redirect loop.
+#
+# The Homebrew CurlDownloadStrategy does not store cookies, so this subclass
+# adds the -c flag and a temporary cookie storage file.
+#
+# This relies on internal details of the CurlDownloadStrategy, so expect it to
+# break.
```

Comment:
` https://mirrors.kernel.org/debian/pool/main/d/delta/delta_2006.08.03.orig.tar.gz ` won't mess you around.